core/server/common/src/main/java/alluxio/master/journal/raft/RaftJournalConfiguration.java

```diff
@@ -53,6 +58,30 @@ public static RaftJournalConfiguration defaults(ServiceType serviceType) {
         .setPath(new File(JournalUtils.getJournalLocation().getPath()));
   }
 
+  /**
+   * @param clusterAddresses addresses of all nodes in the Raft cluster
+   * @return true if clusterAddress have an object of same ip with local Address, otherwise false
+   */
+  private boolean checkContainIp(List<InetSocketAddress> clusterAddresses) {
+    boolean exist = false;
+    String clusterItemIp;
+    String localAddressIp =  NetworkAddressUtils.getLocalIpAddress((int) ServerConfiguration
+        .global().getMs(PropertyKey.NETWORK_HOST_RESOLUTION_TIMEOUT_MS));
+    try {
+      for (InetSocketAddress addr : clusterAddresses) {
+        clusterItemIp = InetAddress.getByName(addr.getHostName()).getHostAddress();
+        if (clusterItemIp.equals(localAddressIp)) {
+          exist = true;
```

Comment:
remove `exist` variable and directly return true here and return false in exception and in the end of the method block?

Reply:
+1